edb/server/protocol/auth_ext/ui/__init__.py

```diff
@@ -707,8 +707,9 @@ def render_password_reset_email(
     app_name: Optional[str] = None,
     logo_url: Optional[str] = None,
     dark_logo_url: Optional[str] = None,
-    brand_color: Optional[str] = "#007bff",
+    brand_color: Optional[str] = "007bff",
 ) -> multipart.MIMEMultipart:
+    brand_color = brand_color if brand_color is not None else "007bff"
```

Comment:
Nitpick: I guess the brand color should be const somewhere so we don't have to repeat it.

Reply:
oh, good call, I can do that now.